generate_documentation.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
The current script that generates the Doxygen test results is [`Scripts/developer_scripts/run_doxygen_testsuite`](https://github.com/CGAL/cgal/blob/760076510cf8c7514f85d2dceb6123fc557479b7/Scripts/developer_scripts/run_doxygen_testsuite).